addons/udes_stock/models/stock_picking.py

```diff
@@ -214,11 +214,14 @@ def create(self, vals):
 
         Checking date_done field if is set to a value but picking is not done.
         """
-        picking = super().create(vals)
-        if not picking.backorder_id:
-            picking.write({"u_original_picking_id": picking.id})
-        picking.check_date_done()
-        return picking
+        pickings = super().create(vals)
+        for picking in pickings:
+            if vals.get("origin"):
+                picking.message_post(body=f"<ul><li>{picking._fields['origin'].string}: {vals.get('origin')}</li></ul>")
```

Comment:
q: can't we just add `track_visibility="onchange"` to the `origin` field? Or does that not go into enough detail?

Reply:
Changed